Resources/Prototypes/_Harmony/Entities/Objects/Weapons/Guns/Basic/staffs.yml

```
@@ -0,0 +1,21 @@
+﻿- type: entity
+  name: staff of slipping
+  parent: WeaponStaffBase
+  id: WeaponStaffSlipping
+  description: For when you need a get-away route.
```

The ordering is incorrect
```suggestion
  parent: WeaponStaffBase
  id: WeaponStaffSlipping
  name: staff of slipping
  description: For when you need a get-away route.
```